app/src/features/authentication/signIn/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { Formik } from 'formik';
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import ModalCheKoV from '../../../components/Modal';
+import FormInput from '../../../components/FormInput';
+import * as TRANSLATIONS from '../../../translations';
+import { useDispatch } from 'react-redux';
+import { loginCustomer } from '../../../redux/auth/modules';
+
+type Props = {
+  shouldRender: boolean;
+  onModalClose: () => void;
+  onSignUpLinkClicked: () => void;
+};
+
+
+const SignIn = ({ shouldRender, onModalClose, onSignUpLinkClicked }: Props) => {
```

Comment:
aj ako te ne mrzi stavi da se folderi za ove feature isto zovu velikim slovima, nekako ima vise smisla

Reply:
ne znam dal mene windows jebe sa imenima opet (desavalo mi se) al meni se i dalje folderi zovu malim slovima

Reply:
Mislim da je to. Meni je sve ok u vscode-u